src/Data/Distributive.purs

```diff
@@ -11,6 +11,14 @@ import Data.Newtype (unwrap)
 -- |   of containers.
 -- | - `collect` is the dual of `traverse` - it traverses an arbitrary
 -- |   collection of values.
+-- |
+-- | Laws:
+-- |
+-- | - `distribute = collect id`
+-- | - `distribute <<< distribute = id`
+-- | - `collect f = distribute <<< map f`
+-- | - `map f = runIdentity <<< collect (Identity <<< f)`
```

Comment:
Good catch. My suspicion is that the potential users of this lib will be familiar with the `unwrap` idiom.